### ISSUE TITLE
Android changing State not reflected in music control

### ISSUE THREAD

**Issue by SteffeyDev** [tanguyantoine/react-native-music-control#54]
Using this code works perfectly on iOS, but on Android when the audio is played, the button in the pull down music controller does not change to pause, the seeker bar does not reflect progress (or anything for that matter), and the artwork is just a large play icon:

`MusicControl.enableControl('play', true);`
`MusicControl.enableControl('pause', true);`
`MusicControl.enableControl('seek', true);`

`MusicControl.setNowPlaying({
             title: title,
             artwork: require('../Images/musicIcon.png'), 
             artist: artist,
             album: album,
             duration: duration // (Seconds)
           });`

When pause tapped in app:
`MusicControl.updatePlayback({
          state: MusicControl.STATE_PAUSED,
          elapsedTime: this.state.currentTime
        });`

When play tapped in app:
`MusicControl.updatePlayback({
          state: MusicControl.STATE_PLAYING,
          elapsedTime: this.state.currentTime
        });`

The artist, title, etc information is updated on both Android and iOS, but that is about all that is working on Android.  Am I doing something wrong or is this a legit bug?

**Comment by Guichaguri** [comment]
Your code should work, I've tested states before and they work as expected.
Can you send screenshots and logcat logs?

**Comment by SteffeyDev** [comment]
![android music screenshot](https://cloud.githubusercontent.com/assets/10255587/24520744/8b9fbace-1557-11e7-8ef5-b55bd09d3324.png)

I have attached a screenshot, and blocked out large portions for privacy's sake.  The title, artist, and album do work, though I did block them as well.  What is not working is the album artwork (I used require, no errors in the logs generated), the play button here should be paused (it simply continually sends the play trigger), and the bar (I think) should show the time and allow me to seek (as specified).  Note that because the same code works in iOS, I know that the JS in my code is correct for the most part.

Not quite sure what logs you want, as there is no output in the logs pertaining to react-native-music-control or anything related.  Thanks for any help
